codestodians/komlie-api
# [Feature]: Allow the addition of quantity to an item added to cart

- Allow for specifying quantity of item to be updated
- update tests

#### What does this PR do?
- Includes the possible addition of the `quantity` field to the `CreateCartItem` mutation

#### Description of Task to be completed?
- Refactor logic for creating cart item to allow for optional inclusion of quantity

#### How should this be manually tested?
- Run the `createCartItem` mutation specifying a `quantity` value or not. Check DB to see changes.

#### What are the relevant pivotal tracker stories?
- [#167107470](https://www.pivotaltracker.com/story/show/167107470)
